covid_graphs/covid_graphs/country_graph.py

```diff
@@ -9,10 +9,14 @@
 from flask import Flask
 import numpy as np
 from pathlib import Path
+from typing import List
 
-from .country_report import Country, CountryReport
+from . import predictions
+from .predictions import CountryPrediction
+from .country_report import CountryReport
 from .formula import ATG_formula, EvaluatedFormula, Formula, XAxisType
 
+
 PREDICTION_DATE = '2020-03-29'
```

Comment:
This constant needs to go away, but I'm fine doing that in another PR. Please add a TODO.
```suggestion
```